components/dashboard/src/user-settings/Preferences.tsx

```diff
@@ -72,6 +89,43 @@ export default function Preferences() {
                         </p>
                     </div>
                 </div>
+
+                <h3 className="mt-12">Inactivity Timeout </h3>
+                <p className="text-base text-gray-500 dark:text-gray-400">
+                    Workspaces will stop after a period of inactivity without any user input.
+                </p>
+                <div className="mt-4 max-w-xl">
+                    <h4>Default Workspace Timeout</h4>
+
+                    {!allowConfigureWorkspaceTimeout && (
+                        <Alert type="message">
+                            Upgrade organization <a href="/billing">billing</a> plan to use a custom inactivity timeout.
```

Comment:
Should be using a `<Link to"" />` instead of an `<a>` so we don't do full page reloads on navigation.